study-builder/studies/rgp/enrollment.conf

```diff
@@ -8535,133 +746,26 @@
-                  "templateType": "TEXT",
-                  "templateText": "$secure_yes",
-                  "variables": [
-                    {
-                      "name": "secure_yes",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.secure_yes} },
-                        { "language": "es", "text": ${i18n.es.secure_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              },
-              {
-                "stableId": "participate_yes",
-                "optionLabelTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$participate_yes",
-                  "variables": [
-                    {
-                      "name": "participate_yes",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.participate_yes} },
-                        { "language": "es", "text": ${i18n.es.participate_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              },
-              {
-                "stableId": "delete_yes",
-                "optionLabelTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$delete_yes",
-                  "variables": [
-                    {
-                      "name": "delete_yes",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.delete_yes} },
-                        { "language": "es", "text": ${i18n.es.delete_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              }
-            ],
-            "validations": [
-              {
-                "ruleType": "NUM_OPTIONS_SELECTED",
-                "minSelections": 3,
-                "maxSelections": 3,
-                "hintTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$secure_required",
-                  "variables": [
-                    {
-                      "name": "secure_required",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.conditions_required} },
-                        { "language": "es", "text": ${i18n.es.conditions_required} }
-                      ]
-                    }
-                  ]
-                }
-              }
-            ]
-          }
+          "question": ${_questions.agreement}
+        },
+
+        //Questions no longer used
+        {
```

Comment:
I would consider moving this question above or below the current ethnicity question.  It could also be reasonable to leave it here or put it just above the agreement checkboxes, but I think this was just an earlier version of the ethnicity question.

Reply:
This question is not visible to users, so I think it's fine wherever we put it. Leaving it here for now.

Reply:
Ah!  Okay